added info to substep (#6109)
open-cluster-management/rhacm-docs: clusters/cluster_lifecycle/cim_create_cli.adoc

<12> Choose one of the following supported CPU architectures: x86_64, aarch64, ppc64le, or s390x. The default value is x86_64.
<13> Causes the image service to always serve the iPXE script when set to the default value of `DiscoveryImageAlways` and when you are using iPXE to boot. As a result, the host boots from the network discovery image. Setting the value to `BootOrderControl` causes the image service to determine when to return the iPXE script, depending on the host state, which causes the host to boot from the disk when the host is provisioned and is part of a cluster.
<14> Allows modifying the kernel arguments for when the Discovery Image boots. Possible values for `operation` are `append`, `replace`, or `delete`.
<15> A PEM-encoded X.509 certificate bundle, usually needed if the hosts are in a network with a re-encrypting man-in-the-middle (MITM) proxy, or if the hosts need to trust certificates for other purposes, such as container image registries. Hosts discovered by your `InfraEnv` trust the certificates in this bundle. Clusters created from the hosts discovered by your `InfraEnv` also trust the certificates in this bundle.

> <16> The Red Hat CoreOS image version to use for your `InfraEnv`. Make sure the version refers to the OS image specified in the `AgentServiceConfig.spec.osImages`, and that `OSImageVersion` matches an {ocp-short} version in the OS images list. You cannot specify `OSImageVersion` and `ClusterRef` at the same time. By default, each release has a specific set of Red Hat CoreOS image versions that are relevant to the next release. Therefore, if you try to add another version, and the Red Hat CoreOS image version does not exist by default, then you must manually add the version. To learn more about adding versions, see _Enabling the central infrastructure management service_. 

Reviewer: I think we want to let the user know that they need to use a version that either they specify in AgentServiceConfig or one that exists by default. Attempted to reword to say that: 

```suggestion
<16> The Red Hat CoreOS image version to use for your `InfraEnv`. Make sure the version refers to the OS image specified in either the `AgentServiceConfig.spec.osImages` or in the default OS images list. Each release has a specific set of Red Hat CoreOS image versions. The `OSImageVersion` must match an {ocp-short} version in the OS images list. You cannot specify `OSImageVersion` and `ClusterRef` at the same time. If you want to use another version of the Red Hat CoreOS image that does not exist by default, then you must manually add the version by specifying it in the `AgentServiceConfig.spec.osImages`. To learn more about adding versions, see _Enabling the central infrastructure management service_. 
```

Feel free to reword!
